### ISSUE TITLE
Is Azure Liveness Detection WebView Supported yet on the mobile?

### ISSUE THREAD

**Issue by marounelkadyNetiks** [Azure-Samples/azure-ai-vision-sdk#122]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> I am trying to include a Azure Liveness Detection Web View in the mobile application built in .NET MAUI, I had issues before and I have been informed that the web view is not yet supported but it was planned for October 2024. I wanted to check if this feature is now available.
If there is any documentation regarding this topic is also highly appreciated.
Thanks!

**Comment by diljale** [comment]
Yes WebView is now supported with latest version of Web SDK. Let us know if you encounter any issues with the same.
